server/src/main/java/org/apache/uniffle/server/NettyDirectMemoryTracker.java

```diff
@@ -68,6 +74,9 @@ public void start() {
             ShuffleServerMetrics.gaugeUsedDirectMemorySize.set(usedDirectMemory);
             ShuffleServerMetrics.gaugeAllocatedDirectMemorySize.set(allocatedDirectMemory);
             ShuffleServerMetrics.gaugePinnedDirectMemorySize.set(pinnedDirectMemory);
+            if (nettyServerEnabled) {
+              shuffleBufferManager.setUsedMemory(pinnedDirectMemory);
```

Comment:
Emm... It's not a good design that use the scheduled thread to update usedMem, which is not determined.